tests/test_rlp_reader.py

```diff
@@ -0,0 +1,131 @@
+#! pytest
+
+import rlp
+from web3 import Web3
+
+
+def test_int_to_rlp_item(test_rlp_reader_contract):
+    """test conversion function from integer to RLPItem"""
```

Comment:
I am slightly confused by RLP, but I don't believe the function `toRLPItem` converts integer to RLPItem as it takes as a parameter `RLP encoded bytes` and that is indeed what you give it (`rlp_encoded_item`)

Reply:
The `toRlpItem` function which I am testing is used for creating an internal data struct `RLPItem`. So basically it takes and already rlp encoded item and converts it to its internally used format. I will adjust the comment to make it more clear.